apps/src/componentLibrary/typography/typography.module.scss

```diff
@@ -256,6 +269,13 @@ em strong,
   @include strong-em;
 }
 
+.em .extra-strong,
```

Comment:
I think this needs to be `.em.extra-strong` without the space for this to work, otherwise it just applies the `.em` italic style.

Reply:
Same applies to line 267 w/ `.em.strong`

Reply:
No, this is intended. To be semantically correct - you should not put two className on one element.
Instead you should use both em and strong tags to make everything more clear semantically (+ I guess it also improves accessability).


Use:
`<EmText><StrongText> Some emphasized strong text here </StrongText></EmText>`

DON'T use:
`<p class="em strong"> Some custom styled text goes here </p>`

Reply:
In that case can we just remove these classes and the one below it: `.extra-strong .em`? We already have these: `em .extra-strong, .extra-strong em`. (And same with `.em .strong,
.strong .em`)

I'm confused why we have these classes at all. I'm not sure if this works differently for React, but I've always just used the semantic HTML tags and don't use the `.em` or `.strong` classes.

Reply:
Agree that it looks messy, but I need it for react implementation to cover all possible variants. 
If on Pegasus you only need semantic tags - I'm ok with Pegasus version not using class names for em/strong, but for react it's better to keep it as is

Reply:
Ok that makes sense if you use the classes in React — I'm going to add the code that makes them work in Pegasus, because what's here wasn't working on Pegasus.